### ISSUE TITLE
keep static from component into container

### ISSUE THREAD

**Issue by pvinis** [zxbodya/rx-react-container#6]
hello.
i have a component with `static navigationOptions = { bla: true, blu: false }`.
then i have a controller with nothing for now, and the container that is combining the props.
then in do `connect(container)(component)`.

now i have the problem where the `navigationOptions` are not there any more in the component returned from `connect`. is there a way to keep that?

**Comment by olessavluk** [comment]
Hello @pvinis,
you can manually assign props to the resulting component, something like:
```js
const Screen = connect(controller)(Component);
Screen.propTypes = { ... };
Screen.navigationOptions = { ... };

export default Screen;
```

Or there is also a library just for that (I have not used it personally) - https://github.com/mridgway/hoist-non-react-statics

**Comment by pvinis** [comment]
thanks @olessavluk. I actually made a pr using hoist.

**Comment by zxbodya** [comment]
merged, see 0.6.1.
thanks for contribution 👍
